topic-configuration/README.adoc

```diff
@@ -696,5 +698,124 @@ h|Kafka property name
++
+--
+*** If {product} found *either* a value or key schema that matches your topic (but not both), the *Schemas* tab displays `No matching schema` next to the schema type that it couldn't find.
++
+To create this type of schema in your {registry} instance, click the question mark icon. In the resulting pop-up window, copy the required naming format, and click *Go to Service Registry instance*.
+
+*** If {product} found *no* schemas that match your topic, the *Schemas* tab displays `No matching schema exists for the selected instance`.
++
+For the type of schema that you want to associate with your topic, copy the required naming format, and click *Go to Service Registry instance*.
+--
++
+The {registry} section of the web console opens with your {registry} instance selected.
+
+. In your {registry} instance, to create a new schema, click *Upload artifact*.
+. In the `ID of the artifact` field, paste the naming format that you previously copied. You must use this naming format so that the new schema matches your Kafka topic.
++
+NOTE: To match your topic, the schema ID must be in the format of `_<topic-name>_-value`, or  `_<topic-name>_-key`.
+
+. When you have finished uploading a new schema, in the {product} web console, click *Streams for Apache Kafka*. Navigate to the *Schemas* tab for your topic, as you did previously.
+. Select the same {registry} instance that you selected previously.
++
+The *Schemas* tab now shows the name of the matching schema that you uploaded.
+. To see details for the schema, or to manage it, click *View details*.
+
+[role="_additional-resources"]
+.Additional resources
+* https://access.redhat.com/documentation/en-us/red_hat_openshift_service_registry/[OpenShift Service Registry documentation^]
+
+[id="proc-creating-registry-instance-and-matching-schema-for-topic_{context}"]
+== Creating a new registry instance and matching schema for a topic
+
+[role="_abstract"]
+The following procedure shows how to use {product} web console to select a Kafka topic, and then create a new {registry} instance with a value or key schema that matches the topic.
+
+Alternatively, to learn how to check an _existing_ {registry} instance for schemas that match a topic, see {base-url}{topic-config-url}#proc-checking-schema-matches-for-topic_{context}[Checking existing schema matches for a topic].
+
+.Prerequisites
+* You're logged in to the {product} web console.
+* You've created a Kafka instance with at least one Kafka topic in {product}. To learn how to do this, see {base-url}{getting-started-url}[Getting started with {product-long}^].
+* You understand how to create a {registry} instance and upload a schema to be used by client applications. To learn how to do this, see {base-url}getting-started-service-registry/README.adoc[Getting started with OpenShift Service Registry^].
+//TO DO:  Add attribute for SR getting started guide
```

Comment:
You can use the {getting-started-service-registry-url} attribute for the SR guide

Reply:
Thanks! I don't think this existed when I started out.

Reply:
Done